src/test/scala/uk/gov/hmcts/ccd/util/CcdTokenGenerator.scala

```diff
@@ -36,19 +92,11 @@ object CcdTokenGenerator extends PerformanceTestsConfig with SpringApplicationCo
     token
   }
 
-  private def roleFor(url: String) = {
-    val result = "caseworker" + parseJurisdiction(url).map(j => s"-$j").getOrElse("")
-    println(s"role used for user token generation: $result")
-    result
-  }
-
-  private def parseJurisdiction(url: String) = {
-    if (!url.contains("jurisdictions")) {
-      None
-    } else {
-      val jurisdiction = url.split("jurisdictions/")(1).split("/")(0)
-      Some(jurisdiction.toLowerCase)
-    }
+  private def parseUrlParameters(url: String) = {
+    url.split("&").map(v => {
```

Comment:
I assume you want to return the query params as  a Map. 
I'd suggest we use this little library: https://github.com/lemonlabsuk/scala-uri#get-query-string-parameters
it could benefit us in other places as well

Reply:
done